src/big/big_float.cr

```diff
@@ -217,11 +243,21 @@ struct BigFloat < Float
     mpf
   end
 
+  # Returns an inspected representation of `self`.
+  #
+  # ```
+  # BigFloat.new("0.123456789101101987654321").inspect # => 0.123456789101101987654321_big_f
+  # ```
   def inspect(io)
     to_s(io)
     io << "_big_f"
   end
 
+  # Returns a string representation of `self`.
```

Comment:
Specify base 10.

We really should support other bases here, but thats another issue.

Reply:
Would you like a `# TODO: support other bases` here ?

Reply:
@RX14 Actually on reflection does it really make much sense in talking about float representation in anything other than base 10, and by extension should I maybe revert the change to specify base 10 ? Certainly there doesn't appear to be any use or mention of alternate bases in Float32 / Float64 for instance. Perhaps you were thinking of BigInt? 
```
+  # Returns a string representation (base 10) of `self`.
+  #
+  # ```
+  # BigFloat.new("0.123456789101101987654321").to_s # => 0.123456789101101987654321
+  # ```
   def to_s(io : IO)
```